src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,42 @@
+import { FC } from "react";
+import { Link } from "../../interfaces/Links";
+import classes from "./Footer.module.css";
+
+const linksChild = [
+  { id: "home", title: "Главная", href: "#" },
+  { id: "aboutUs", title: "О нас", href: "#" },
+  { id: "menu", title: "Меню", href: "#" },
+  { id: "delivery", title: "Доставка", href: "#" },
+  { id: "contacts", title: "Контакты", href: "#" },
+  { id: "coffeeShops", title: "Кофейни", href: "#" },
+  { id: "newItems", title: "Новинки", href: "#" },
+  { id: "coffee", title: "Кофе", href: "#" },
+  { id: "desserts", title: "Десерты", href: "#" },
+  { id: "syrup", title: "Строп", href: "#" },
+];
+
+export const Footer: FC = () => {
+  return (
+    <footer className={classes.footer}>
+      <div className={classes.content}>
+        <h1 className={classes.title}>coffee</h1>
+        <div className={classes.linksWrapper}>
+          {linksChild.map(({ title, href, id }: Link) => (
+            <a key={id} href={href} className={classes.link}>
+              {title}
+            </a>
+          ))}
+        </div>
+        <div className={classes.contacts}>
+          <button className={classes.makeOrder}>Сделать заказ</button>
+          <a href="telephone" className={classes.phone}>

```

Comment:
https://www.elegantthemes.com/blog/wordpress/call-link-html-phone-number#:~:text=Adding%20an%20HTML%20Phone%20Number%20Call%20Link%20to%20your%20Website&text=Href%3Dtel%3A%20creates%20the%20call,the%20number%20it%20will%20call.